package.json

```diff
@@ -72,7 +72,7 @@
 		"buffer": "^6.0.3",
 		"eventemitter3": "^5.0.1",
 		"readable-stream": "^4.5.2",
-		"utilium": "^1.0.0"
+		"utilium": "file:../utilium"
```

Comment:
This needs to be fixed.

Reply:
Changed to `^1.1.1` that includes the required declaration.